investment-app/TRINV.Api/TRINV.InvestTrackApplication/Program.cs

```diff
@@ -31,6 +36,13 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+builder.Services.AddMediatR(cfg =>
```

Comment:
@glachezar  This is not the best way to initialize mediator in the program.cs. Find better way

Reply:
Is the example below the best way of registering of MediatR?
builder.Services.AddMediatR(cfg => cfg.RegisterServicesFromAssemblyContaining<Program>();

Reply:
Yes  !
![image](https://github.com/MihaelPavlov/Angular_Projects/assets/57910640/93b5f986-fd88-4900-b381-02932573421e)